File: Makefile.go
#
# Makefile
#
all: usage

usage:
	@echo ""
	@echo "usage: make [edit|build|run]"
	@echo ""

PGM=vector
#PGM=vision

edit e:
	vi $(PGM).go

edit-make em:
	vi Makefile

edit-readme er:
	vi README.md

build b: $(PGM).go
	go build -o $(PGM) $(PGM).go

run r:
	go run $(PGM).go

view v:
	eog *.jpg *.png

clean:
	rm -f $(PGM)

File: Makefile.py
#
# Makefile by stoney, sikang99@gmail.com
#
all: usage

usage:
	@echo ""
	@echo "usage: make [edit|build|run]"
	@echo ""

PGM=test


install-package pk:
	sudo pip install numpy scipy matplotlib
	sudo pip install pillow lxml jupyter protobuf
	sudo pip install tensorflow keras
	#sudo pip install tensorflow-gpu

edit e:
	vi $(PGM).py

run r:
	python $(PGM).py

